app/models/concerns/galtersufia/generic_file/invenio_resource_type_mappings.rb

```diff
@@ -0,0 +1,180 @@
+        "Blogs": "Social Media Posting",
+        "Case Reports": "Case Report",
+        "Catalogs": "Catalog",
+        "Clinical Trial": "Clinical Trial",
+        "Collected Correspondence": "Collection; Correspondence",
+        "Conference Proceeding": "Conference Proceeding",
+        "Congresses": "Congress",
+        "Dataset": "Dataset",
+        "Dictionary": "Book",
+        "Drawings": "Drawing",
+        "Editorial Article": "Journal article, Editorial",
+        "Electronic Supplementary Materials": "Chart; Table; Dataset; Bibliography; Technical Documentation",
+        "Ephemera": "Ephemera",
+        "Evaluation Studies": "Evaluation Study",
+        "Examination Questions": "Examination Questions",
+        "Exhibitions": "Exhibition",
+        "Fictional Works": "Fictional Work",
+        "Forms": "Form",
+        "Guideline": "Guideline; Study Design",
+        "Handbooks": "Handbook",
+        "Historical Article": "Other Article",
+        "Image": "Pictorial Work, Photograph",
+        "Introductory Journal Article": "Journal Article; Other Article",
+        "Journal Article": "Journal Article",
+        "Journal": "Journal",
+        "Lecture Notes": "Lecture Notes",
+        "Lectures": "Lecture",
+        "Lecture": "Letter; Correspondence",
+        "Manuscript": "Manuscript; Preprint",
+        "Maps": "Map",
+        "Masters Thesis": "Masters Thesis",
+        "Meeting Abstracts": "Meeting Abstract",
+        "News": "News; Newsletter",
+        "Other": "Other; Other Article",
+        "Periodicals": "Magazine; Newsletter",
+        "Photographs": "Photograph",
+        "Pictorial Works": "Pictorial Work",
+        "Poetry": "Poetry",
+        "Portraits": "Portrait",
+        "Postcards": "Postcard",
+        "Poster": "Conference Poster; Other",
+        "Presentation": "Presentation",
+        "Programmed Instruction": "Presentation",
+        "Programs": "Program",
+        "Project": "", #TODO: Check with Sara if this is necessary. According to note from box csv file there are only
+                       # two projects one that is private and one for the collection
+        "Report": "Report",
+        "Research Paper": "Report; Review; Other Article",
+        "Resource Guides": "Resource Guide",
+        "Review": "Review",
+        "Software or Program Code ": "Other; Software; Technical Documentation",
+        "Speech": "Lecture; Presentation; Sound (Recording)",
+        "Statistics": "Statistics",
+        "Study Design": "Study Design",
+        "Tables": "Table",
+        "Technical Report": "Technical Documentation",
+      }.with_indifferent_access
+
+      # Mapping of IRDM sub type / specific resource type to IRDM resource type / category
+      IRDM_RESOURCE_TYPES = {
+        "Collection": "Archival Items",
+        "Ephemera": "Archival Items",
+        "Exhibition": "Archival Items",
+        # "Physical Object": "Archival Items",
```

Comment:
Commented out fields are invenio specific types -> invenio types that did not match any DH values or records. In other words unless someone were to add something to DH that would then map to one of these commented out fields then we would have a single record using it. I figure it's not a bad idea to keep them here since I don't want to retype this again (after accidentally screwing something up with it and having to audit it x_x). If it's really causing problems we can remove these lines.

Reply:
Thanks for adding all of this! I appreciate the work you did here. Good idea adding and commenting out all of the other fields that are only in invenio.